src/pyOpenMS/addons/MatrixDouble.pyx

```diff
@@ -1,62 +1,48 @@
 from Matrix cimport *
-
+cimport numpy as np
 
 # continue with extra code if needed
 
-    def get_matrix(self):
-        """Cython signature: numpy_matrix get_matrix()
+    def get_matrix_as_view(self):
+        """Cython signature: numpy_matrix get_matrix_as_view()
```

Comment:
I think, ideally, if you have time, since pyx docstrings are directly taken over, we should not fake a nonsense "cython signature" here. Just try to type the function signature and write the typed function signature into the first line of the docstring (to be parsed by sphinx).
After that, document what the function does.